### DISCUSSION TITLE
I have found how to unpin tiles in the Windows 10 Start Menu

### DISCUSSION THREAD

**Discussion by Kalenias** [valinet/ExplorerPatcher#2185] [Show and tell]
I have discovered this when i thought the unpin feature was fixed, when I tested unpinning a folder it let me unpin it, then I discovered that if you group 2 or more tiles in a folder you can unpin them and save space from your start menu.

![image](https://github.com/valinet/ExplorerPatcher/assets/100655927/2cf8a728-3691-420c-b757-8eb351784976)

I hope it works for you 😊

**Comment by milestprower92** [comment]
Go To C:\Windows\SystemApps\MicrosoftWindows.Client.Core_cw5n1h2txyewy rename StartMenu.dll and restart Explorer

**Comment by pyrates999** [comment]
known issues: https://github.com/valinet/ExplorerPatcher/discussions/1679

Please post windows version and build.  You can find this by running winver.
Please post explorer patcher version.

Please include what settings you changed in explorer patcher properties application.
